docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md

```diff
@@ -34,10 +34,10 @@ The following diagram shows the recommended components for a Great Expectations
 
 Relevant documentation for the components can also be found here:
 
-- [How to configure an Expectation store to use GCS](https://docs.greatexpectations.io/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs)
-- [How to configure a Validation Result store in GCS](https://docs.greatexpectations.io/docs/guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_gcs)
-- [How to host and share Data Docs on GCS](https://docs.greatexpectations.io/docs/guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs)
-- Optionally, you can also use a [Secret Manager for GCP Credentials](https://docs.greatexpectations.io/docs/guides/setup/configuring_data_contexts/how_to_configure_credentials)
+- [How to configure an Expectation store to use GCS](/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs)
+- [How to configure a Validation Result store in GCS](/docs/guides/setup/configuring_metadata_stores/how_to_configure_a_validation_result_store_in_gcs)
+- [How to host and share Data Docs on GCS](/docs/guides/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs)
+- Optionally, you can also use a [Secret Manager for GCP Credentials](/docs/guides/setup/configuring_data_contexts/how_to_configure_credentials)
```

Comment:
😭  thank you for these fixes

Reply:
Just for future reference, are links not supposed to go to `https://docs.greatexpectations.io/docs/`?